backend/rest/authRoutes.ts

```diff
@@ -167,4 +168,23 @@ authRouter.post(
   },
 );
 
+/* Reset password through a "Forgot Password" option */
+authRouter.post(
```

Comment:
Just for conversation:

```
authRouter.post(
  "/resetPassword/:email",
  isAuthorizedByEmail("email"),
  async (req, res) => {
    try {
      await authService.resetPassword(req.params.email);
      res.status(204).send();
    } catch (error: unknown) {
      res.status(500).json({ error: getErrorMessage(error) });
    }
  },
);
```

Can we reuse this isAuthorizedByEmail() function for added security? I see that it's apart of starter code so it may not be applicable to us?